package.json

```diff
@@ -32,17 +32,15 @@
     "test": "vitest run --coverage",
     "test:watch": "vitest"
   },
-  "bin": {
-    "mx-widget-sdk-setup": "dist/cli/setup.js"
-  },
   "dependencies": {
     "@mxenabled/widget-post-message-definitions": "^1.4.0",
     "react-native-base64": "^0.2.1",
     "url": "^0.11.0"
   },
   "peerDependencies": {
     "react": "*",
-    "react-native": "*"
+    "react-native": "*",
+    "react-native-webview": "*"
```

Comment:
Can we set a version here? I know 11 will cause issues. The latest version is 13.16.0, what if we set this to 13.*?